atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -238,8 +239,25 @@ def _make_labels(self):
 
     def do_worker_build(self, release, cluster_info):
         build = None
+        task_id = None
+        fs_result = self.workflow.prebuild_results.get(AddFilesystemPlugin.key)
+        if fs_result is not None:
+            try:
+                fs_task_id = fs_result['filesystem-koji-task-id']
+            except KeyError:
+                self.log.error("%s: expected filesystem-koji-task-id in result",
+                               AddFilesystemPlugin.key)
+            else:
+                try:
+                    task_id = int(fs_task_id)
+                except ValueError:
+                    self.log.error("invalid taskID %r", fs_task_id, exc_info=1)
+
         try:
             kwargs = self.get_worker_build_kwargs(release, cluster_info.platform)
+            self.log.debug("%s: setting filesystem_koji_task_id to %d",
+                           AddFilesystemPlugin.key, task_id)
+            kwargs['filesystem_koji_task_id'] = task_id
```

Comment:
Yeah, passing it and setting it in there would be more consistent, didn't look closely enough at that. Change pushed that does that. I'm not sure what you mean by "if it's truthy" though. Looks like we should throw an exception if we get a non-numeric task ID, not sure what other sort of verification we can do here on the fly?